docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -15,6 +15,104 @@ ms.custom: devx-track-azurecli
 
 # [Current release notes](#tab/azure-cli)
 
+## October 20, 2020
```

Comment:
The date is not correct, and it should be October 13, 2020

Reply:
The CI uses the next Tuesday by default. However, we were blocked last Friday.